DM-2193: update documentation for migrate operation (#972)
SoftwareAG/c8y-docs: content/protocol-integration/lwm2m-bundle/connector-device.md

### Migration of the LWM2M devices

Starting from version 10.15.0, the new device registration for LWM2M is introduced.
The LWM2M devices which were created earlier than v10.15, or if new LWM2M devices are created using the "General bulk registration" method, then those devices must be migrated
to the new structure using `migrateLwm2mDevices` command for the tenant. The migrate operation will try to migrate all devices and their existing client registration objects to 
the new format in the database. It will be backwards compatible since it retains old fragments. 

> Arguments `-d` or `--devices` followed by the device managed object ID(s) can be used to migrate specific device managed objects. To skip the

Reviewer: ```suggestion
The argument `-d` or `--devices` followed by a list of managed object IDs can be used to migrate specific device managed objects. To skip the
```
